File: FrontEnd (C#)/SoftProgModel/GestMaterial/Libro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftProgModel.GestMaterial
{
    public class Libro : MaterialBibliografico
    {
        private string ISBN;
        private string edicion;

        public Libro(): base()
        {
        }

        public Libro(string ISBN, string edicion, int idMaterial, int numero_copias, string titulo, int anho_publicacion, 
            int numero_paginas, EstadoMaterial estado, string clasificacion_tematica, bool activo, string idioma, TipoMaterial tipo):
            base(idMaterial, titulo, anho_publicacion, numero_paginas, estado, clasificacion_tematica, activo, idioma, tipo)
        {
            this.ISBNP = ISBN;
            this.Edicion = edicion;
        }

        public string ISBNP { get => ISBN; set => ISBN = value; }
        public string Edicion { get => edicion; set => edicion = value; }
    }
}

File: BackEnd (Java)/UtilsArmyModel/src/main/java/pe/edu/pucp/utilsarmy/gestion_de_material/model/Libro.java
package pe.edu.pucp.utilsarmy.gestion_de_material.model;

public class Libro extends MaterialBibliografico {
    private String ISBN;
    private String edicion;
    
    public Libro(){
        super();
    }

    public Libro(String ISBN, String edicion, int numero_copias, String titulo, int anho_publicacion, int numero_paginas, EstadoMaterial estado, String clasificacion_tematica, boolean activo, String idioma, String editoriales) {
        super(titulo, anho_publicacion, numero_paginas, estado, clasificacion_tematica, activo, idioma, editoriales);
        this.ISBN = ISBN;
        this.edicion = edicion;
    }

    public String getISBN() {
        return ISBN;
    }

    public void setISBN(String ISBN) {
        this.ISBN = ISBN;
    }

    public String getEdicion() {
        return edicion;
    }

    public void setEdicion(String edicion) {
        this.edicion = edicion;
    }
 
}
